components/script/script_thread.rs

```diff
@@ -3669,6 +3678,15 @@ impl ScriptThread {
                 };
                 document.dispatch_composition_event(composition_event);
             },
+
+            GamepadEvent(gamepad_event) => {
+                let window = match self.documents.borrow().find_window(pipeline_id) {
+                    None => return,
```

Comment:
Let's do as the others `None => return warn!("Message sent to closed pipeline {}.", pipeline_id),`